core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -90,6 +92,54 @@ package object config {
     .toSequence
     .createWithDefault(Nil)
 
+  // Blacklist confs
+  private[spark] val BLACKLIST_ENABLED =
+    ConfigBuilder("spark.blacklist.enabled")
+      .booleanConf
+      .createOptional
+
+  private[spark] val MAX_TASK_ATTEMPTS_PER_EXECUTOR =
+    ConfigBuilder("spark.blacklist.task.maxTaskAttemptsPerExecutor")
+      .intConf
+      .createWithDefault(1)
```

Comment:
Note sure I follow your nit here, this is how the ConfigEntries work, you define the config, type, default, etc.  if you have issues with it in general it should be a separate jira against the Config stuff.